### ISSUE TITLE
Improve name parsing and add address parsing

### ISSUE THREAD

**Issue by aafaq-hassan-confiz** [OmkarPathak/pyresparser#11]
Excellent work!

Can you please try to improve name parsing?

Can you please add address parsing too?

Thanks.

**Comment by OmkarPathak** [comment]
@aafaq-hassan-confiz yes. I'm constantly trying to improve the parsing. Address parsing will be available in next release. Thanks for the feedback :smile:

**Comment by aafaq-hassan-confiz** [comment]
It parses phone number, but many times phone number was not parsed, please add improving phone parsing in your backlog.

When will you publish latest changes to https://pypi.org/project/pyresparser?

**Comment by OmkarPathak** [comment]
@aafaq-hassan-confiz soon 😄

**Comment by AliKazim7** [comment]
Hi Omkar excellent work buddy really love this script.
Currently i am working on feature where we can extract complete data from file and display it our web. Till now im able to extract data to my requirement but face error on Extracting name sometime it will give wrong name or something else. Plus is there a way where i can extract only companies name from CV? thanks again for such great script

**Comment by aafaq-hassan-confiz** [comment]
Please publish latest changes to https://pypi.org/project/pyresparser

**Comment by OmkarPathak** [comment]
@aafaq-hassan-confiz new version 1.0.6 is released. Please let me know if you get desired results

**Comment by aafaq-hassan-confiz** [comment]
Thanks @OmkarPathak.  I have noticed in new release that phone parsing has been improved but now phone number is not being completely parsed, last 2 or more digits are being discarded from parsed phone number.

**Comment by OmkarPathak** [comment]
@aafaq-hassan-confiz I have now added custom regex parsing option. You can add the regex for phone numbers of you country and parser would parse them accordingly

**Comment by aafaq-hassan-confiz** [comment]
@OmkarPathak I have to parse resumes across different countries, so I can't use regex for a single country. So if you could find the current issue and improve your regex then that would be a great help, thanks in advance 😄

**Comment by OmkarPathak** [comment]
@aafaq-hassan-confiz you can try a generic regex. The reason behind I incorporated custom regex was because I couldn't find a generic regex that would parse all the mobile numbers. You can find some generic ones on Stack Overflow. All the best. Till the being can we close this issue?

**Comment by aafaq-hassan-confiz** [comment]
@OmkarPathak this ticket is specifically for improving name parsing and adding address parsing. I think this ticket should remain open till at least address parsing is added.

**Comment by aafaq-hassan-confiz** [comment]
@OmkarPathak when will address parsing be added? Please add address parsing as soon as possible, I need it, thanks.

**Comment by mukulanand8** [comment]
Hey @OmkarPathak First of all, thanks for the script! Super helpful! Regarding name parsing, is it possible to exclude results such as "Curriculum Vitae" and "Resume" and use the next possible name?

**Comment by OmkarPathak** [comment]
@mukulanand8 yes we can. We simply have to filter out those. You can do it by changing [this](https://github.com/OmkarPathak/pyresparser/blob/master/pyresparser/utils.py#L337)

**Comment by sauravi-5** [comment]
Hey @OmkarPathak First things first, great job at the resume parser and thanks for the code. It's of great help. On the future work part of it, how about adding a section for parsing projects and hobbies if any?
